.ebextensions/00_ami.config

```diff
@@ -50,8 +50,8 @@ option_settings:
 
   # Load Balancer security group
   aws:elbv2:loadbalancer:
-    SecurityGroups: ["$SECURITY_GROUP_LOAD_BALANCER"]
-    ManagedSecurityGroup: "$SECURITY_GROUP_LOAD_BALANCER"
+    SecurityGroups: [sg-0563c2648b0b301eb]
+    ManagedSecurityGroup: sg-0563c2648b0b301eb
```

Comment:
We are reverting to https://github.com/ets-berkeley-edu/diablo/blob/5736a28b605fb6f654445e5011df025916a67624/.ebextensions/00_ami.config